packages/mobile/ios/Podfile

```diff
@@ -28,7 +28,7 @@ use_frameworks!
 
 target "celo" do
   pod "CeloBlockchain", :path => "../../../node_modules/@celo/client/CeloBlockchain.podspec", :modular_headers => true
-
+  pod 'clevertap-react-native', :path => '../../../node_modules/clevertap-react-native'
```

Comment:
This shouldn't be needed with autolink.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2012/commits/d2af69ea54a10a7425e5acbed011e464a7d89a5b!